src/components/reference-book/exercise.cjsx

```diff
@@ -14,8 +14,9 @@ ReferenceBookExercise = React.createClass
   displayName: 'ReferenceBookExercise'
   render: ->
     {exerciseAPIUrl} = @props
-    {items} = ReferenceBookExerciseStore.get(exerciseAPIUrl)
-
+    exercise = ReferenceBookExerciseStore.get(exerciseAPIUrl)
+    return null unless exercise
```

Comment:
This removes the "Exercise Missing" text when an exercise is missing.

How about https://github.com/openstax/tutor-js/commit/7ce1035d1f629547a0ff95c64225521fd7af3567#diff-73b870b5a9922ddf9e623aa1807f684a instead?